corehq/apps/repeaters/models.py

```diff
@@ -287,6 +298,37 @@ def get_auth(self):
             return HTTPDigestAuth(self.username, self.password)
         return None
 
+    def fire_for_record(self, repeat_record):
+        headers = self.get_headers(repeat_record)
+        auth = self.get_auth()
+        payload = self.get_payload_and_handle_exception(repeat_record)
+        url = self.get_url(repeat_record)
+        try:
+            response = simple_post(payload, url, headers=headers, timeout=POST_TIMEOUT, auth=auth)
+        except (Timeout, ConnectionError) as error:
+            log_repeater_timeout_in_datadog(self.domain)
+            self.handle_response(RequestConnectionError(error), repeat_record)
```

Comment:
The alternative is to just let the next `except` block handle this type of error, which I think might be fine too. Our custom Repeaters currently log a `RequestConnectionError` to the case, but you could just as easily change those references to `if isinstance(Timeout, ConnectionError)` which is a little more verbose, but remains consistent. 

Or, just have the custom repeaters log every exception to the case, which I'd be in favor of too.